frontend/src/pages/destructionlist/detail/pages/process-review/DestructionListProcessReviewPage.tsx

```diff
@@ -137,6 +146,93 @@ export function DestructionListProcessReviewPage() {
     }));
   }, [reviewItems, zaakReviewStatuses]);
 
+  /**
+   * Get the process review badge for a zaak to know what mutation is proposed.
+   */
+  function getProcessReviewBadge(zaak: Zaak) {
+    const processZaakReviewDetail =
+      processZaakReviewSelectionState?.[zaak.url as string]?.detail;
+    const action = processZaakReviewDetail?.action;
+
+    if (!action) {
+      return (
+        // @ts-expect-error - style props not supported (yet?)
+        <Badge style={{ display: "block" }}>
+          <Solid.QuestionMarkCircleIcon />
+          Geen Mutatie
+        </Badge>
+      );
+    }
+
+    const { archiefactiedatum, selectielijstklasse, comment } =
+      processZaakReviewDetail;
+
+    const getLabel = (value: string | undefined) =>
+      selectieLijstKlasseChoicesMap?.[zaak.url]?.find(
+        (choice) => choice.value === value,
+      )?.label;
+
+    const tooltipData = {
+      archiefactiedatum,
+      oud_archiefactiedatum: zaak?.archiefactiedatum,
+      selectielijstklasse: getLabel(selectielijstklasse),
+      oud_selectielijstklasse: getLabel(zaak.selectielijstklasse),
+      opmerking: comment,
+    };
+
+    // Filtering logic:
+    // 1. Keep 'oud_' values only if the new value exists and is different.
+    // 2. Remove empty, null, or undefined values.
+    // 3. Remove duplicate values.
+    const filteredTooltipData = Object.fromEntries(
+      Object.entries(tooltipData).filter(([key, value], _, arr) => {
+        // 1.
+        if (key.startsWith("oud_")) {
+          const newKey = key.replace("oud_", "");
+          const newValue = tooltipData[newKey as keyof typeof tooltipData];
+
+          return (
+            newValue !== undefined && newValue !== "" && newValue !== value
+          );
+        }
+        // 2.
+        if (value == null || value === "") return false;
+
+        // 3.
+        return !arr.some(
+          ([otherKey, otherValue]) => otherKey !== key && otherValue === value,
+        );
+      }),
+    );
+
+    const tooltipContent =
+      Object.keys(filteredTooltipData).length > 0 ? (
+        <AttributeTable object={filteredTooltipData} />
+      ) : (
+        "Geen wijzigingen"
+      );
+
+    return (
+      <Tooltip content={tooltipContent} size="md">
+        <span>
+          {action === "keep" ? (
+            // @ts-expect-error - style props not supported (yet?)
+            <Badge level="info" style={{ display: "block" }}>
```

Comment:
I think we should make sure the badge is focussable to make the tooltip appear on tab, I think we can do this with tabindex="0"